src/main/java/org/easetech/easytest/internal/DateTimeFormat.java

```diff
@@ -42,7 +42,7 @@
     /**
      * The date/time formats to use as specified by the user or the default formats
      */
-    private String[] dateTimeFormat = {"dd/MM/yy HH:MM:SS" ,"dd/MM/yyyy HH:MM:SS", "dd-MM-yy HH:MM:SS", "dd-MM-yyyy HH:MM:SS"};
+    private String[] dateTimeFormats = {"dd/MM/yy HH:MM:SS" ,"dd/MM/yyyy HH:MM:SS", "dd-MM-yy HH:MM:SS", "dd-MM-yyyy HH:MM:SS"};
```

Comment:
I generally agree with this change but it has to be a collective change not an isolated change. Thus, in the DateTimeFormat.java class, dateFormat, timeFormat should also be renamed dateFormats and timeFormats.
In addition, the name of the getters and setters should also be changed appropriately.